client/node/package.json

```diff
@@ -1,19 +1,20 @@
 {
-  "name": "@suga/client",
-  "version": "1.0.0",
-  "description": "suga runtime client",
+  "name": "@nitric/suga-client",
```

Comment:
Will need to change the generated code to use @nitric/suga-client with this changing.